lincbrain/utils.py

```diff
@@ -610,10 +610,10 @@ def _get_instance(
             f" please contact that server's administrators: {e}"
         )
     our_version = Version(__version__)
-    if our_version < minversion:
-        raise CliVersionTooOldError(our_version, minversion, bad_versions)
-    if our_version in bad_versions:
-        raise BadCliVersionError(our_version, minversion, bad_versions)
```

Comment:
Similar to https://github.com/lincbrain/linc-cli/pull/2/files#r1406776675 -- `versioneer` needs to be refined